src/Provider/GoogleMaps/Model/SubLocalityLevelCollection.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\GoogleMaps\Model;
+
+use Geocoder\Exception\CollectionIsEmpty;
+use Geocoder\Exception\InvalidArgument;
+use Geocoder\Exception\OutOfBounds;
+
+/**
+ * Class SubLocalityLevelCollection is used only for GoogleMap provider, contains functions for working with arrays of SubLocalityLevel
+ */
+final class SubLocalityLevelCollection implements \IteratorAggregate, \Countable
```

Comment:
This looks exactly the same as AdminLevelCollection. Maybe we should use that instead